packages/bson-bench/package.json

```diff
@@ -17,7 +17,7 @@
   "author": "The MongoDB NodeJS Team <dbx-node@mongodb.com>",
   "license": "Apache-2.0",
   "dependencies": {
-    "bson": "^6.1.0"
+    "bson": "4.7"
```

Comment:
Note: this change was meant to be merged into main prior to this, but was used when collecting results for NODE-4004.

Reply:
That's confusing, what's the bson library package used for inside the tooling?

Reply:
More to the point, how can this tooling be cross version and cross library compatible if it depends on a specific library at a specific version?

Reply:
Isn't this the "setup" bson used to generate docs and such? why wouldn't this be the latest?

Reply:
This bson version needs to be v4 for compatibility with bson-ext. It's only used [here](https://github.com/mongodb-js/dbx-js-tools/blob/c9c18764d3ec4b462d198bc6312cf6b707809811/packages/bson-bench/src/base.ts#L44-L45) in the event that we don't have access to EJSON to parse our test documents to an object as is the case in bson-ext. bson-ext only works with documents created with bson v4.

Reply:
If that's the case, can we add a comment to that extent in the line you linked for clarity?

Reply:
Thanks, all good on my end